tests/Database/QueryTest.php

```diff
@@ -329,7 +339,7 @@ public function testHaving()
 			->having('balance', '<=', 100)
```

Comment:
Again, to ensure that the result is not the same by accident:

```suggestion
			->having('balance', '<=', 70)
```